### PR TITLE
Improve python CLI quickstart documentation

### LINKED ISSUE #306: Show the python package in the doc
it will be nice to show the python package with a simple install and howto, it is not very clear from the doc, if there is a ptyhon package at all ?

### PR DESCRIPTION
**Pull request by kevinjqliu**
Fixes #306

Add instructions to run the CLI in the main `README.md` file. 
Aligned instructions for all 3 readme files (`README.md` and `tpchgen-cli/README.md`)
Put `uvx` first, then `pip install`

### CODE REVIEW

File: benchmarks/BENCHMARKS.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ This directory contains a set of scripts to generate TPCH data at various scale
 
 # IO Throughput Limitations
 
-[`tpchgen-cli`](tpchgen-cli/README.md) is the fastest TPCH generator we know of
+[`tpchgen-cli`](../tpchgen-cli/README.md) is the fastest TPCH generator we know of
```
Comment (kevinjqliu) [inline benchmarks/BENCHMARKS.md:7]: not related to this PR but codex found this broken link

File: tpchgen-cli/README.md [hunk 2]
```diff
@@ -19,16 +19,16 @@ project details
...
 ```shell
-pip install tpchgen-cli
+uvx tpchgen-cli parquet -s 1 --output-dir /tmp/tpch
 ```
 
-## Install via `uv`
+## Install with `pip`
 
 ```shell
-uv tool install tpchgen-cli
```
Comment (kevinjqliu) [inline tpchgen-cli/README.md:31]: i like `uvx` more than `uv tool`

Comment (alamb) [reply tpchgen-cli/README.md:31]: uvx is so cool 

```rust
andrewlamb@Andrews-MacBook-Pro-3 Downloads % uvx tpchgen-cli
nation     [████████████████████████████]   Progress: 100%                      region     [████████████████████████████]   Progress: 100%                      part       [████████████████████████████]   Progress: 100%                      supplier   [████████████████████████████]   Progress: 100%                      partsupp   [████████████████████████████]   Progress: 100%                      customer   [████████████████████████████]   Progress: 100%                      orders     [████████████████████████████]   Progress: 100%                      lineitem   [████████████████████████████]   Progress: 100%
```

Comment (alamb) [reply tpchgen-cli/README.md:31]: Posted about this on the socials
https://x.com/andrewlamb1111/status/2073013111141855510
https://bsky.app/profile/andrewlamb1111.bsky.social/post/3mpqik2nrh22j

File: README.md [hunk 3]
```diff
@@ -20,7 +20,26 @@ Blazing fast [TPCH] benchmark data generator, in pure Rust with zero dependencie
 
 ## Try it now
 
-The easiest way to use this software is via the [`tpchgen-cli`] tool.
+Try with `uvx`:
```
Comment (alamb) [inline README.md:23]: I recommend also adding an animated gif for extra attention yeeting. Here is one I made for consideraton

Image

**Review by alamb — APPROVED** [review]
Love it! Thank you @kevinjqliu

**Comment by kevinjqliu** [comment]
Looks amaing, thanks @alamb ! 
https://github.com/clflushopt/tpchgen-rs/blob/27933fcb5aae913aec1f550ae9de0e9945b470d3/README.md

Screenshot 2026-07-03 at 9 52 27 AM

**Comment by kevinjqliu** [comment]
we probably want to update these performance numbers: https://github.com/clflushopt/tpchgen-rs#performance

SF1 seems way faster now:
```
➜  ~ time uvx tpchgen-cli parquet -s 1 -o /tmp/tpchgen-cli
nation     [████████████████████████████]   Progress: 100%                      
region     [████████████████████████████]   Progress: 100%                      
part         [████████████████████████████]   Progress: 100%                      
supplier   [████████████████████████████]   Progress: 100%                      
partsupp [████████████████████████████]   Progress: 100%                      
customer [████████████████████████████]   Progress: 100%                      
orders     [████████████████████████████]   Progress: 100%                      
lineitem   [████████████████████████████]   Progress: 100%
uvx tpchgen-cli parquet -s 1 -o /tmp/tpchgen-cli  
4.80s user 0.25s system 400% cpu 1.259 total
```

**Comment by kevinjqliu** [comment]
Ran into a weird github CI edge case... 

The TPC-DS Conformance tests are marked as required but only triggers for changes related to `'tpcdsgen/**'` or `'.github/**'`
https://github.com/clflushopt/tpchgen-rs/blob/fe5ba9f9c97d14f4f008e7565043425ef07690b5/.github/workflows/tpcdsgen-conformance.yml#L9-L13

This PR was stuck since it cannot trigger those CI checks.
I removed the `paths` filter so the CI check will run to unblock this PR. Will take a look at a better long term fix

**Comment by kevinjqliu** [comment]
Oh I see, the "required" checks are define in github setting. These two are marked as required:
```
Conformance Tests (Java)
Conformance Tests (C dsdgen)
```

It might be a good idea to not make them always required, so `tpcdsgen-conformance.yml` can run for `tpcdsgen` changes only

@clflushopt wdyt?

**Comment by kevinjqliu** [comment]
tracking the CI issue in https://github.com/clflushopt/tpchgen-rs/issues/324
